tests/functional/full-load-ok-escaping/expected/data/out/tables/output.csv

```diff
@@ -1,7 +1,7 @@
 "1",""""""""
 "2","'''"
 "3","```"
-"4","1234úěščřžýáíé+!@#$%^&*()"
+"4","1234áúívw"
```

Comment:
Tento test na kodovanie prechadzal, aj ked vlastne CLIENT_LOCALE bolo blbo.
Nasa testovacia db ma locale: `8859-1`, PHP pouziva `utf-8`. 

V `8859-1` niektory znaky z test, napr `ž` vobec neexistuju + kazdy znak je jeden byte (do db sa to vklada v `EscapingTableTrait.php` vid nizsie).

V `utf-8` je znak jeden, alebo 2 byte, ... `ž` su 2 byte. 

A teda stalo sa to, ze Informix zobral `ž` ako 2 byte v `8859-1`, teda 2 rozne znaky. A tak ich ulozil.
Pri nacitani extractorom sa spravil opacny proces, a v PHP sa to opat spojilo.

Pritom to bolo v databaze ulozene blob, no v teste to vyzeralo OK.

Po fixnuti CLIENT_LOCALE tento test v povodnej podobe spadne, ... lebo `ž` sa neda ulozit, co je tak spravne.
Preto je tu teraz uz iba zopar znakov, ... ktore su platne v `8859-1` a DB ich vie ulozit spravne.